opts/envdir.go

```diff
@@ -0,0 +1,56 @@
+package opts
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+/*
+Read in a directory with files acting as environment variables.
+*/
+func ParseEnvDir(directory string) ([]string, error) {
+	dir, err := ioutil.ReadDir(directory)
+	if err != nil {
+		return []string{}, err
+	}
+
+	lines := []string{}
+	for _, info := range dir {
+		variable := info.Name()
+		// ignore directories and hidden files
+		if !info.IsDir() && variable[0] != '.' {
+			if strings.ContainsAny(variable, whiteSpaces) {
```

Comment:
Yeah, not entirely sure what all of the restrictions there are, or if we should care. But as is, I don't want to address this with this pull request. Is that fair?